### PR TITLE
fix: enable panic backtraces

### PR DESCRIPTION
**Pull request by rchincha**
We expect panics in the server/datapath to be few and far between. So the backtraces are more valuable now.

**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/api/controller.go [hunk 1]
```diff
@@ -117,8 +117,9 @@ func (c *Controller) Run() error {
 
 	engine.Use(
 		SessionLogger(c),
-		handlers.RecoveryHandler(handlers.RecoveryLogger(c.Log),
```
Comment (andaaron) [inline pkg/api/controller.go:120]: Why remove the handlers.RecoveryLogger(c.Log)?

Comment (rchincha) [reply pkg/api/controller.go:120]: Keeping that causes panic backtrace to not show.

Comment (andaaron) [reply pkg/api/controller.go:120]: So where should it be shown, in stdout or in the logfile?
I assume in whatever place is configured via zot config, so where is that reference to the logger in this specific case?

Comment (andaaron) [reply pkg/api/controller.go:120]: Shouldn't changing true to false be enough to solve the issue?

Comment (rchincha) [reply pkg/api/controller.go:120]: Doesn't appear to be enough.

**Review by andaaron — APPROVED** [review]

**Pull request merged**